soccer/src/soccer/ui/main_window.cpp

```diff
@@ -49,6 +52,7 @@ void calcMinimumWidth(QWidget* widget, const QString& text) {
     widget->setMinimumWidth(rect.width());
 }
 
+// TODO: Set up the UI objects and bind them to methods
```

Comment:
is this later done?

Reply:
Yes, starting on line 1174. Forgot to remove this comment.